src/config/client.ts

```diff
@@ -1,10 +1,9 @@
 import axios from 'axios';
 
-const baseUrl = process.env.NEXT_PUBLIC_API_URL || 'http://localhost:8001/api/v1';
+const backendURL = process.env.NEXT_PUBLIC_API_URL;
 
-const axiosClient = axios.create({
-  baseURL: baseUrl,
-});
+const axiosClient = axios.create({ baseURL: `${backendURL}`, timeout: 20000, withCredentials: true});
+const config = {headers: {'Content-Type': 'application/json', 'Access-Control-Allow-Origin': '*'}};
```

Comment:
I don't think `config` is being used anywhere.  Can you remove it?

Reply:
It’s not used yet